src/components/property-detail/PropertyDetail.js

```diff
@@ -0,0 +1,23 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from 'react';
+import { getPropertyDetail as get } from '../../mockBackend/fetch';
+
+
+const PropertyDetail = () => {
+  const { propertyId } = useParams();
+  
+  const [listing, setListing] = useState(null);
```

Comment:
There is a warning we need to fix before merging

> Line 9:10:  'listing' is assigned a value but never used  no-unused-vars

Can be as simple as render the listing id in line 18